.gitignore

```diff
@@ -148,3 +148,7 @@ rsync.*
 
 # VIM YCM
 .ycm_extra_conf.py
+
```

Comment:
These changes should go in your local `Trilinos/.git/info/exclude` file and not in the repository's `.gitignore`.  `.git/info/exclude` is the equivalent of a `.gitignore` file, but just for your own local repository&mdash;it won't be committed/tracked/pushed, so it's for files you want to ignore that aren't necessarily files that everyone who clones the repo would want to ignore.